djangocms_versioning/test_utils/factories.py

```diff
@@ -158,7 +158,7 @@ class PageContentFactory(factory.django.DjangoModelFactory):
     changed_by = FuzzyText(length=12)
     in_navigation = FuzzyChoice([True, False])
     soft_root = FuzzyChoice([True, False])
-    template = FuzzyText(length=12)
+    template = 'page.html'
```

Comment:
Why was the template changed?

Reply:
Templates was changed as it must exist to be able to use it. `FuzzyText` templates does not exist.

Reply:
Thinking about this, this should still be randomly generated. If you want to use the page then you set that parameter when you use it. Most tests don't require it so it's right that it's just a random value. The internal codebase does this a little better where it selects from a list of templates.

Reply:
Factories should have sane defaults. FuzzyText is not a sane default considering that a correct value is a template that exists. 

I would agree that a randomly picked template that exist is better than a single value that exist. 

Generating text that has no meaning, when in fact the underlying value should point to something else is not something that is desirable. So I disagree with you on this point.